code/modules/clothing/under/miscellaneous.dm

```diff
@@ -370,3 +370,12 @@ obj/item/clothing/under/genericr
 		bio = 100,
 		rad = 30
 	)
+
+//Onestar
+//obj/item/clothing/under/onestar
```

Comment:
Due to this line being commented the values below are assigned to the tuxedo above, so... Did you even test it?

Reply:
fixed it and tested it. issue is resolved now